### PR TITLE
[v2.5.6] Release system-charts

### PR DESCRIPTION
**Pull request by ryansann**

### CODE REVIEW

**Review by luthermonson — DISMISSED** [review]

File: charts/rancher-eks-operator/1.0.600/questions.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.6-alpha1
```
Comment (deniseschannon) [inline charts/rancher-eks-operator/1.0.600/questions.yaml:1]: Do older versions need a rancher max version?

File: charts/rancher-monitoring/v0.1.5/questions.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.9-rc1
```
Comment (aiyengar2) [inline charts/rancher-monitoring/v0.1.5/questions.yml:1]: Related: https://github.com/rancher/system-charts/pull/422#discussion_r585915174

Comment (deniseschannon) [inline charts/rancher-monitoring/v0.1.5/questions.yml]: Same comment as within 2.4 check.

Shouldn't this be 2.4.14-rc1 and there be a max version of the monitoring version v0.1.4, so there is only 1 monitoring chart for k8s 1.16?

File: charts/rancher-monitoring/v0.2.1/questions.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.4-rc1
```
Comment (deniseschannon) [inline charts/rancher-monitoring/v0.2.1/questions.yml]: Shouldn't this be 2.5.6-rc1 and the monitoring v0.2.0 should have a max Rancher version so only 1 Rancher version is available for this line of monitoring?

**Review by deniseschannon — CHANGES_REQUESTED** [review]
Why are there changes in logging 0.2.3 when logging 0.2.3 is already released?

File: charts/rancher-monitoring/v0.1.5/questions.yml (hunk 2 above)
Comment (aiyengar2) [reply charts/rancher-monitoring/v0.1.5/questions.yml]: https://github.com/rancher/system-charts/pull/423

Comment (aiyengar2) [reply charts/rancher-monitoring/v0.1.5/questions.yml]: To clarify based on offline discussion, there will still be 2 charts for Monitoring in k8s 1.16 even after the above PR is resolved: 0.1.5 and 0.2.1

However, currently 0.1.4 and 0.2.0 are also visible in 2.5.6 for k8s 1.16 users, which is what the above PR resolves by adding a rancher_max_version to those charts.

File: charts/rancher-eks-operator/1.0.600/questions.yaml (hunk 1 above)
Comment (cbron) [reply charts/rancher-eks-operator/1.0.600/questions.yaml:1]: It picks the latest one, so not really. cc @rmweir

File: charts/rancher-istio/1.5.901/questions.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+labels:
+  rancher.istio.v1.5.901: 1.5.9
+rancher_min_version: 2.4.5-rc1
```
Comment (cbron) [inline charts/rancher-istio/1.5.901/questions.yaml:3]: We don't need to bump this and cap the previous version ? I'm not actually familiar with changes. Typically unless its security or a very bad bug we don't restrict.

**Review by cbron — COMMENTED** [review]
1 question, needs logging fix, then looks ok

**Review by deniseschannon — DISMISSED** [review]

File: charts/rancher-istio/1.5.901/questions.yaml (hunk 4 above)
Comment (deniseschannon) [reply charts/rancher-istio/1.5.901/questions.yaml:3]: I think it depends on the chart type as whether or not you can pick your own version. 

Does Istio allow you to pick what chart you want to deploy?

Comment (cbron) [reply charts/rancher-istio/1.5.901/questions.yaml:3]: Yes

Comment (deniseschannon) [reply charts/rancher-istio/1.5.901/questions.yaml:3]: I've approved the changes for the release, but we could discuss how much we want to limit Istio per release. 

In the past, we've limited to only 1 version based on the support matrix but not sure how much that matters/impacts for Istio.

Comment (cbron) [reply charts/rancher-istio/1.5.901/questions.yaml:3]: Ok, I was unaware of the support matrix limit. I had previously thought we left multiple versions available if they were just patch or minor version changes. 
This change is not an upstream version change, just a bug on our side: https://github.com/rancher/rancher/issues/30964
Given that, I kinda think we should cap the last one.

Comment (cbron) [reply charts/rancher-istio/1.5.901/questions.yaml:3]: We have decided to cap 1.5.900, putting PR for that in now.

**Pull request merged**